README.md

```diff
@@ -8,30 +8,42 @@ The following instructions are tested on Linux and OSX systems.
 
 Compiler requirement:
 
-* gcc 4.8 (required to be ABI compatible with libarrow).
+* g++-4.8 or
+* g++-5.4
 
 ## Get dependencies
 
 **Note**: This repo uses submodules. Make sure you cloned recursively:
-```
+
+```bash
 git clone --recurse-submodules git@github.com:gpuopenanalytics/libgdf.git
 ```
+
 Or, after cloning:
-```
+
+```bash
 cd libgdf
 git submodule update --init --recursive
 ```
 
-It is recommended to setup a conda environment for the dependencies.
+Since cmake will download and build Apache Arrow you may need to install Boost C++:
+
+```bash
+$ sudo apt-get install libboost-all-dev
+```
```

Comment:
You should also be able to get the boost deps from conda-forge as well.

Reply:
Additionally I don't think we can assume this is going to be running on a debian machine.

Reply:
Thanks for the feedback. I made a new PR https://github.com/gpuopenanalytics/libgdf/pull/64